frontend/src/features/admin-form/common/AdminViewFormService.ts

```diff
@@ -196,3 +196,65 @@ export const submitStorageModeFormPreview = async ({
     submissionContent,
   ).then(({ data }) => data)
 }
+
+/**
+ * Submit an email mode form in preview mode using fetch
+ * TODO(#5826): Fallback using Fetch. Remove once network error is resolved
+ */
+export const submitEmailModeFormPreviewWithFetch = async ({
+  formFields,
+  formLogics,
+  formInputs,
+  formId,
+}: SubmitEmailFormArgs): Promise<SubmissionResponseDto> => {
+  const filteredInputs = filterHiddenInputs({
+    formFields,
+    formInputs,
+    formLogics,
+  })
+  const formData = createEmailSubmissionFormData(formFields, filteredInputs)
+
+  const response = await fetch(
+    `${API_BASE_URL}/${ADMIN_FORM_ENDPOINT}/${formId}/preview/submissions/email`,
+    {
+      method: 'POST',
+      body: formData,
+    },
+  )
+  return response.json()
+}
+
+/**
+ * Submit a storage mode form in preview mode using fetch
+ * TODO(#5826): Fallback using Fetch. Remove once network error is resolved
+ */
+export const submitStorageModeFormPreviewWithFetch = async ({
+  formFields,
+  formLogics,
+  formInputs,
+  formId,
+  publicKey,
+}: SubmitStorageFormArgs): Promise<SubmissionResponseDto> => {
+  const filteredInputs = filterHiddenInputs({
+    formFields,
+    formInputs,
+    formLogics,
+  })
+  const submissionContent = await createEncryptedSubmissionData(
+    formFields,
+    filteredInputs,
+    publicKey,
+  )
+
+  const response = await fetch(
+    `${API_BASE_URL}/${ADMIN_FORM_ENDPOINT}/${formId}/preview/submissions/encrypt`,
+    {
+      method: 'POST',
+      body: JSON.stringify(submissionContent),
+      headers: {
+        'Content-Type': 'application/json',
```

Comment:
is there a reason why we use `JSON.stringify` and specify the Content-Type headers here (vs `submitEmailModeFormPreviewWithFetch` and the `submitStorageModeFormPreview` axios call)?

Reply:
For extra info here:
* For email mode, we generate a [FormData](https://developer.mozilla.org/en-US/docs/Web/API/FormData) object, which is understood by both Axios and `fetch()`, so we don't need to set custom headers.
* For Storage mode, we supply a POJO object. Axios automatically applies JSON-encoding to it, and sets the header accordingly, while when we use `fetch()`, we have to do it ourselves (see [docs here](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#uploading_json_data))